src/frontend/src/components/public/PIASubHeader/index.tsx

```diff
@@ -59,6 +60,19 @@ function PIASubHeader({
     setShowModal(false);
   };
 
+  const showSubmitButton = () => {
```

Comment:
I think these if else for different roles and statuses is all over the place. We may need to add it to 'status' state machine at some point, under utlils/status file.